controllers/etcdpeer_controller.go

```diff
@@ -176,8 +232,16 @@ func (r *EtcdPeerReconciler) Reconcile(req ctrl.Request) (ctrl.Result, error) {
 
 	log.V(2).Info("Found EtcdPeer", "name", peer.Name)
 
+	// Apply defaults in case a defaulting webhook has not been deployed.
+	peer.Default()
+
+	created, err := r.maybeCreatePvc(ctx, &peer)
```

Comment:
do we want to leave an option for non-persistent clusters using the container's filesystem?

Reply:
I'd prefer if the operator didn't have too many branches and different sets of expected resources.
A non-persistent cluster can be acheived by creating hostpathVolumes or localPersistentvolumes backed by tmpfs.